core/modules/filters/has.js

```diff
@@ -45,7 +45,7 @@ exports.has = function(source,operator,options) {
 				if(tiddler && $tw.utils.hop(tiddler.fields,operator.operand) && !(tiddler.fields[operator.operand] === "" || tiddler.fields[operator.operand].length === 0)) {
 					results.push(title);
 				}
-			});
+			});				
```

Comment:
Seems to be not intended?

Reply:
Indeed! We really need to add a step to test.sh that will fail if there are any JS source code lines with trailing whitespace...